src/calculate-fps.js

```diff
@@ -0,0 +1,35 @@
+export default (frames, visibilityEvents) =>
+  1000 / (
+    (
+      (frames[frames.length - 1] - frames[0]) -
+      getHiddenDelta(visibilityEvents)
+    ) /
+    (frames.length - 1)
+  )
+
+export const getHiddenDelta = (visibilityEvents) =>
+  visibilityEvents
+  .reduce((deltas, [visible, timestamp]) => {
+    const last = deltas.slice(-1)[0]
+    const remaining = deltas.slice(0, -1)
+    if (visible && !last) { return deltas }
+
+    if (!visible) {
```

Comment:
I don't follow what this is doing. Why is it duplicating the last timestamp when the tab becomes visible?